apps/src/templates/Certificate.jsx

```diff
@@ -0,0 +1,26 @@
+import React, { PropTypes, Component } from 'react';
+import i18n from "@cdo/locale";
+import color from "../util/color";
+
+const styles = {
+  heading: {
+    color: color.teal,
+    width: '100%'
+  }
+};
+
+export default class Certificate extends Component {
+  static propTypes = {
+    completedTutorialType: PropTypes.oneOf(['applab', 'newMinecraft', 'oldMinecraft', 'other']).isRequired
```

Comment:
We will soon have three Minecraft tutorials.  How does "new" and "old" relate, and can we possibly be more specific?